packages/caliper-fabric/test/connector-versions/v1/WalletFacade.js

```diff
@@ -0,0 +1,141 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+chai.should();
+const mockery = require('mockery');
+
```

Comment:
Add the following line 
```
/* eslint-disable require-jsdoc */
```
Then we can remove all the JSDoc comments that had to be provided for the StubWallet. 
I plan to do the same for the V2 wallet